src/main/java/org/folio/innreach/controller/InnReachTransactionController.java

```diff
@@ -60,4 +61,11 @@
     var transactions = transactionService.getAllTransactions(offset, limit, parameters);
     return ResponseEntity.ok(transactions);
   }
+
+  @Override
+  @PutMapping("/inn-reach/transactions/{id}")
+  public ResponseEntity<InnReachTransactionDTO> updateInnReachTransaction(@PathVariable UUID id, InnReachTransactionDTO transaction) {
+    var innReachTransaction = transactionService.updateInnReachTransaction(id, transaction);
+    return ResponseEntity.ok(innReachTransaction);
+  }
```

Comment:
Usually we don't return anything with update endpoints. You can check an example of existing CRUD controllers to see how we handle update 
https://github.com/folio-org/mod-inn-reach/blob/e931b68064a8d3da25c28a681e6acf28015c32e3/src/main/java/org/folio/innreach/controller/CentralServerController.java#L63-L68

Reply:
thanks for help